src/main/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcPageHeadingLabel.scala

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmrc.hmrcfrontend.views.config
+
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+
+object HmrcPageHeadingLabel {
```

Comment:
I guess my only question is whether it needs the Hmrc prefix. We didn't for StandardPhaseBanner but then perhaps we should have? 🤷

Reply:
yeah, I wasn't sure either way, I defaulted to using it because it seems like the recommendation is slightly different to the govuk one https://design-system.service.gov.uk/styles/typography/captions-inside/index.html